ci/scripts/lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eux
+
+pushd dp-legacy-redirector
+  go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.3
```

Comment:
this is the latest (I think)
```suggestion
  go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.57.2
```